### PR TITLE
Increase Lawnchair's visibility on funding

### PR DESCRIPTION
**Pull request by validcube**
## Description

Introduce Open Collective entry to Lawnchair's About page and an sponsor label to GitHub Repository

https://opencollective.com/lawnchair

- [x] GitHub Repository funding label
- [x] In-apps About section

## Type of change

✅ General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by SuperDragonXD — COMMENTED** [review]
Thanks for the draft PR. While I initially planned on doing this, I appreciate you stepping up and making a PR for this.

We'd appreciate it more if you'd also add a donate button in Lawnchair preferences.

File: lawnchair/src/app/lawnchair/ui/preferences/about/About.kt [hunk 1]
```diff
@@ -298,8 +298,29 @@ fun About(
...
+            ClickablePreference(
+                label = stringResource(id = R.string.translate),
+                onClick = {
+                    val webpage = Uri.parse(CROWDIN_URL)
+                    val intent = Intent(Intent.ACTION_VIEW, webpage)
+                    if (intent.resolveActivity(context.packageManager) != null) {
+                        context.startActivity(intent)
+                    }
+                },
+            )
```
Comment (SuperDragonXD) [inline lawnchair/src/app/lawnchair/ui/preferences/about/About.kt]: Seems like you duplicated the "Translate" entry. Could you remove this first?

**Review by SuperDragonXD — CHANGES_REQUESTED** [review]

File: lawnchair/src/app/lawnchair/ui/preferences/about/About.kt [hunk 2]
```diff
@@ -298,8 +298,29 @@ fun About(
                     }
                 },
             )
+            ClickablePreference(
+                label = stringResource(id = R.string.translate),
```
Comment (validcube) [inline lawnchair/src/app/lawnchair/ui/preferences/about/About.kt]:
```suggestion
                label = stringResource(id = R.string.donate),
```

**Review by validcube — COMMENTED** [review]
Fix skill issue from android studio

File: lawnchair/src/app/lawnchair/ui/preferences/about/About.kt (hunk 1 above)
Comment (validcube) [reply lawnchair/src/app/lawnchair/ui/preferences/about/About.kt]: Whoops, I meant to do R.string.donate

Fixed

File: lawnchair/src/app/lawnchair/ui/preferences/about/About.kt [hunk 3]
```diff
@@ -298,8 +298,29 @@ fun About(
...
+            ClickablePreference(
+                label = stringResource(id = R.string.donate),
+                onClick = {
+                    val webpage = Uri.parse(CROWDIN_URL)
+                    val intent = Intent(Intent.ACTION_VIEW, webpage)
+                    if (intent.resolveActivity(context.packageManager) != null) {
+                        context.startActivity(intent)
+                    }
+                },
+            )
```
Comment (validcube) [inline lawnchair/src/app/lawnchair/ui/preferences/about/About.kt]:
```suggestion
```

**Review by validcube — COMMENTED** [review]
Remove dups... Another skill issue

File: lawnchair/src/app/lawnchair/ui/preferences/about/About.kt (hunk 1 above)
Comment (validcube) [reply lawnchair/src/app/lawnchair/ui/preferences/about/About.kt]: Actually now fixed (removed)

**Review by SuperDragonXD — APPROVED** [review]

**Pull request merged**